types/src/transaction/authenticator.rs

```diff
@@ -1127,13 +1149,24 @@ impl EphemeralSignature {
         &self,
         message: &T,
         public_key: &EphemeralPublicKey,
+    ) -> Result<()> {
```

Comment:
i assume this `EphemeralSignature::verify` is used in VM too...
you sure the new version is equivalent to the old version?

Reply:
forgot to revert here too - fixed